instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/db/SqlStatementSanitizerTest.java

```diff
@@ -308,6 +309,12 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) th
               expect("update \"my table\" set answer=?", "UPDATE", "my table")),
           Arguments.of("update /*table", expect("UPDATE", null)),
 
+          // Call
+          Arguments.of("call test_proc()", expect("CALL", "test_proc")),
+          Arguments.of("call test_proc", expect("CALL", "test_proc")),
+          Arguments.of("call next value in hibernate_sequence", expect("CALL", null)),
```

Comment:
(not sure what happens, but may be a good test also)
```suggestion
          Arguments.of("call next value in hibernate_sequence", expect("CALL", null)),
          Arguments.of("select next value in hibernate_sequence", expect("SELECT", null)),
```

Reply:
@trask Yep I did add it, its in the //Select section of the test cases